### PR TITLE
Wepack setup

### PR DESCRIPTION
**Pull request by JakeOrel**

### CODE REVIEW

**Comment by JakeOrel** [comment]
@AmiraVomir Set up the webpack and bundle let me know what you think

**Comment by AmiraVomir** [comment]
@JakeOrel add bundle.js to .gitignore

**Comment by JakeOrel** [comment]
@AmiraVomir added bundle to my gitignore

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      exclude: /(node_modules)/,
+      use: {
+        loader: "babel-loader",
+        options: {
+          presets: ["@babel/preset-env", "@babel/preset-react"]
+        }
+      }
+    }],
+  }
+}
```
Comment (carlitoswillis) [inline webpack.config.js:24]: new line?

**Pull request merged**